huaweicloud/services/dws/resource_huaweicloud_dws_workload_plan_stage.go

```diff
@@ -0,0 +1,361 @@
+package dws
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var defaultErrMsg = "The resource does not exist or is illegal, please contact technical support for assistance"
```

Comment:
```suggestion
var defaultErrMsg = "the resource is illegal, please contact technical support"
```
It's not possible to return a `resource not found` error at the reference codes where you return this error code, whether create or update.

Reply:
done